fetcher/util.go

```diff
@@ -13,22 +14,50 @@ import (
 	"github.com/htcat/htcat"
 	"github.com/inconshreveable/log15"
 	"github.com/spf13/viper"
+	"github.com/ulikunitz/xz"
 	"github.com/vulsio/goval-dictionary/util"
 	"golang.org/x/xerrors"
 )
 
+type mimeType int
+
+const (
+	mimeTypeXML mimeType = iota
+	mimeTypeTxt
+	mimeTypeBzip2
+	mimeTypeXz
+	mimeTypeGzip
+)
```

Comment:
Possibly also YAML.
https://github.com/vulsio/goval-dictionary/blob/master/fetcher/alpine.go#L9-L10

It might be a good idea to write the mimeType explicitly.
https://github.com/maito1201/goval-dictionary/blob/support_fedora/fetcher/debian.go#L20-L24